decentraland/unity-renderer
# Feat: UI gradient emotes

## What does this PR change?



This PR adds the custom gradient elements to the emotes wheel selector, which is now animated.

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch=feat/UI-Gradient-Emotes
2. Open the emotes wheel.
3. Put the mouse over an element of the wheel.
4. Verify the selector is visible and animated.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
